src/Registration/pReg/Reg.py

```diff
@@ -44,89 +49,15 @@
 ERROR_CHANNEL = 2
 ALL_CHANNELS = -1
 
-###########################################################
-############ Utilities for internal use only ##############
-
 
-def _setParameter_sirf(hs, set_, par, hv, stack=None):
-    # try_calling(pyreg.cReg_setParameter(hs, set, par, hv))
+def _set_parameter(hs, group, par, hv, stack = None):
```

Comment:
Can't this one be unified (and therefore deleted from here)?

Reply:
this is just a quick fix to sort out the error you hit first

Reply:
which error? My initial error was just that the `select_module.py` and `parameters.py` weren't getting installed.

Reply:
have you fixed this?